fe/fe-core/src/main/java/com/starrocks/connector/iceberg/IcebergConnector.java

```diff
@@ -15,50 +15,92 @@
 
 package com.starrocks.connector.iceberg;
 
+import com.google.common.base.Strings;
+import com.starrocks.common.util.Util;
 import com.starrocks.connector.Connector;
 import com.starrocks.connector.ConnectorContext;
 import com.starrocks.connector.ConnectorMetadata;
 import com.starrocks.connector.HdfsEnvironment;
+import com.starrocks.connector.exception.StarRocksConnectorException;
 import com.starrocks.credential.CloudConfiguration;
 import com.starrocks.credential.CloudConfigurationFactory;
+import org.apache.hadoop.conf.Configuration;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
 import java.util.Map;
 
 import static com.starrocks.catalog.IcebergTable.ICEBERG_CATALOG_LEGACY;
 import static com.starrocks.catalog.IcebergTable.ICEBERG_CATALOG_TYPE;
+import static com.starrocks.catalog.IcebergTable.ICEBERG_IMPL;
+import static com.starrocks.catalog.IcebergTable.ICEBERG_METASTORE_URIS;
 
 public class IcebergConnector implements Connector {
     private static final Logger LOG = LogManager.getLogger(IcebergConnector.class);
-
     private final Map<String, String> properties;
     private final CloudConfiguration cloudConfiguration;
     private final HdfsEnvironment hdfsEnvironment;
     private final String catalogName;
-    private ConnectorMetadata metadata;
+    private final IcebergCatalog icebergNativeCatalog;
+
+    private String metastoreURI;
```

Comment:
metastoreURI/catalogImpl/catalogType/customProperties will be deleted in next pr